introduction.md

```diff
@@ -3,5 +3,7 @@ On my way to becoming a GitHub Master. I love Ruby and JavaScript.
 =======
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?